river/base/transformer.py

```diff
@@ -32,6 +34,28 @@ def learn_one(self, x: dict, **kwargs) -> "Transformer":
         """
         return self
 
+    def learn_many(self, X: pd.DataFrame, **kwargs) -> "Transformer":
```

Comment:
It would be better here to add `MiniBatchTransformer` class (like `MiniBatchClassifier` and `MiniBatchRegressor`). Indeed, by adding this `learn_many` method to `Transformer`, it makes it look like each transformer has a `learn_many` method.

Reply:
Fair